pkg/webui/console/views/device-add/wizard/network-settings-form/index.js

```diff
@@ -38,6 +38,36 @@ import {
 
 import validationSchema from './validation-schema'
 
+/**
+ * Filters available lorawan phy versions by lorawan version.
+ *
+ * @param {number} version - Lorawan version.
+ * @returns {Array} - A list of matching lorawan phy version options based on `version`.
+ */
+const getLorawanPhyByVersion = version => {
+  if (version === 100) {
+    return [{ value: '1.0.0', label: 'PHY V1.0' }]
+  } else if (version === 101) {
+    return [{ value: '1.0.1', label: 'PHY V1.0.1' }]
+  } else if (version === 102) {
+    return [
+      { value: '1.0.2-a', label: 'PHY V1.0.2 REV A' },
+      { value: '1.0.2-b', label: 'PHY V1.0.2 REV B' },
+    ]
+  } else if (version === 103) {
+    return [{ value: '1.0.3-a', label: 'PHY V1.0.3 REV A' }]
+  } else if (version === 104) {
+    return [{ value: '1.0.0', label: 'PHY V1.0' }, { value: '1.0.1', label: 'PHY V1.0.1' }]
```

Comment:
@rvolosatovs please implement or guide @bafonins extending the API for allowing the RP2 format. See referenced issue for the table.

For this PR, we might need to skip this step (1.0.4), as we don't have a valid PHY version to select from.

---

On a side note; any reason not to use `switch`/`case`?